cabinet/cabinet.py

```diff
@@ -56,6 +56,20 @@ def get_all(self):
     def get(self, name):
         return self._vault.get(name)
 
+    def get_by_tags(self, tags=[]):
```

Comment:
You should not use `[]` as default value, instead do:

```python
def get_by_tags(self, tags=None):
    if tags is None:
        tags = []
```

See an explanation here: http://effbot.org/zone/default-values.htm